Extend documentation of the editor (#75)
kbss-cvut/s-pipes-editor-ui: README.md

# SPipes Editor UI

SPipes Editor UI is editor of [SPipes](https://github.com/kbss-cvut/s-pipes) scripts with following features:

> - visualization of the scripts

Reviewer: Maybe the following:

`script visualization`

> - visualization of the scripts

Reviewer: ```suggestion
- script visualization
```

> - visualization of the scripts

Reviewer: ```suggestion
- script visualization
```

>   - exploration of the related filesystem structure where scripts are located

Reviewer: Maybe the following:

`browse the related filesystem structure where the scripts are stored`

>   - exploration of the related filesystem structure where scripts are located

Reviewer: ```suggestion
  - browse the related filesystem structure where the scripts are stored
```

>   - represent the pipelines as graphs

Reviewer: Maybe the following:

`represent pipelines as interactive graphs`

>   - represent the pipelines as graphs

Reviewer: ```suggestion
  - represent pipelines as interactive graphs
```

>   - collapse/expand groups of nodes based on the source file they come from

Reviewer: Maybe the following:

`collapse/expand groups of nodes based on the source file`

>   - collapse/expand groups of nodes based on the source file they come from

Reviewer: ```suggestion
  - collapse/expand groups of nodes based on the source file
```

>   - automatic layout of graph nodes

Reviewer: Maybe the following:

`automatically arrange graph nodes`

>   - automatic layout of graph nodes

Reviewer: ```suggestion
  - automatically arrange graph nodes
```

> - modification of the scripts

Reviewer: Maybe the following:

`script Modification`

> - modification of the scripts

Reviewer: ```suggestion
- script Modification
```

>   - notification if script file changes outside of the editor

Reviewer: Maybe the following:

`receive notifications if a script file changes outside the editor`

>   - notification if script file changes outside of the editor

Reviewer: ```suggestion
  - receive notifications if a script file changes outside the editor
```

>   - custom editors for specific modules (e.g. SPARQL query editor)

Reviewer: Maybe the following:

`use custom editors for specific modules (e.g., an SPARQL query editor)`

>   - custom editors for specific modules (e.g. SPARQL query editor)

Reviewer: ```suggestion
  - use custom editors for specific modules (e.g., an SPARQL query editor)
```

>   - editor to edit ttl directly

Reviewer: Maybe thee following:

`directly edit .ttl files using a built-in editor.`

>   - editor to edit ttl directly

Reviewer: ```suggestion
  - directly edit .ttl files using a built-in editor.
```

> - execution of the scripts

Author: ```suggestion
-  script execution
```

>   - execute scripts with parameters

Reviewer: Maybe the following:

`execute scripts with specified parameters`

>   - execute scripts with parameters

Reviewer: ```suggestion
  - execute scripts with specified parameters
```

>   - explore the history of executions

Reviewer: Maybe the following:

`browse the history of previous executions`

>   - explore the history of executions

Reviewer: ```suggestion
  - browse the history of previous executions
```

>   - explore the history of executions

Reviewer: ```suggestion
  - browse the history of previous executions
```
